src/main/java/org/lsc/beans/syncoptions/PropertiesBasedSyncOptions.java

```diff
@@ -196,6 +196,60 @@ public String getCondition(LscModificationType operation) {
 		}
 		return result;
 	}
+
+	public String getPostHookOutputFormat() {
+		if (conf.getHooks() == null || conf.getHooks().getOutputFormat() == null) {
+			return "";
+		}
+		return conf.getHooks().getOutputFormat();
+	}
+
+	public String getCreatePostHook() {
+		if (conf.getHooks() == null || conf.getHooks().getCreatePostHook() == null) {
+			return "";
```

Comment:
style: I usually prefer returning an Optional<String> to indicate there is a value or not.

But here I did not check what are the consequences, it can be a little more complicated to handle if you are not used to.